src/components/ChannelSearch/ChannelSearch.tsx

```diff
@@ -82,7 +90,7 @@ const UnMemoizedChannelSearch = <
   const { t } = useTranslationContext();
 
   const [query, setQuery] = useState('');
-  const [results, setResults] = useState<Array<UserResponse<Us>>>([]);
+  const [results, setResults] = useState<Array<ChannelOrUserType> | []>([]);
```

Comment:
gonna need generics

Reply:
oh wait, there is generics there. it's of type:

```
| Channel<
      DefaultAttachmentType,
      DefaultChannelType,
      DefaultCommandType,
      DefaultEventType,
      DefaultMessageType,
      DefaultReactionType,
      DefaultUserType
    >
  | UserResponse<DefaultUserType>;
```